README.md

```diff
@@ -20,5 +20,8 @@ List of the [GitHub](https://github.com) usernames of all the people interested
 - [Aya Salama] (https://github.com/Aya-S)
 - [Bhagya C](https://github.com/BhagyaC)
 
-
 This list is not yet complete—we'll keep adding more people.
+
+## Lessons Learned:
+Please use the link below to review or capture key lessons learned as new contributors: <br>
+[Lessons Learned/ Knowledge transfer documentation]( https://github.com/python-sprints/pandas-mentoring/blob/master/LEARNING_POINTS.md)
```

Comment:
probably won't cause trouble, but the space between the link shouldn't probably be there

Reply:
@datapythonista ! Absolutely, I removed the link break and spacing between the link. Thank you!